src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensEditsHandler.cs

```diff
@@ -208,10 +214,7 @@ public static IEnumerable<SequenceEdit> GetEdits(int[] oldSemanticTokens, int[]
                 {
                     var edits = s_instance.GetEdits(
                         oldSemanticTokens, oldSemanticTokens.Length, newSemanticTokens, newSemanticTokens.Length);
-
-                    // By default, edits are returned largest -> smallest index. For computation purposes later on,
-                    // we can want to have the edits ordered smallest -> largest index.
-                    return edits.Reverse();
```

Comment:
@ryanbrandenburg mentioned that `.Reverse()` isn't very performant, so I instead baked this logic into the for-loop earlier in the file.